OpenSim/Common/Exception.cpp

```diff
@@ -58,12 +57,6 @@ exception()
     setMessage(aMsg);
     _file = aFile;
     _line = aLine;
-
```

Comment:
This is also a drive-by edit that was made because I was using regex to flag uses of `assert`, and this one is likely to be a cute developer trick that was probably used a while ago (i.e. before IDEs like Visual Studio had debugger toggles to break on throwing an exception) but has been buried here.